www/js/app.js

```diff
@@ -365,9 +365,10 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     // Top menu :
     $('#btnHome').on('click', function() {
         // Highlight the selected section in the navbar
-        $('#liHomeNav').attr("class","active");
-        $('#liConfigureNav').attr("class","");
-        $('#liAboutNav').attr("class","");
+        document.getElementById('liHomeNav').setAttribute("class", "active");
+        document.getElementById('liConfigureNav').setAttribute("class", "");
+        document.getElementById('liAboutNav').setAttribute("class", "");
+
```

Comment:
Removed :)
Amended the previous commit and force pushed.

Reply:
OK, but JFYI it's generally best not to force push while you're in a PR others are reviewing, because it forces them to reset their copy. It doesn't matter if the commit history gets long, because you'll squash to a single commit before merging to master in any case. 🙂

Reply:
Oh alright. Sorry, I'll keep this in mind :)